examples/echo/main.go

```diff
@@ -107,3 +128,21 @@ func main() {
 	// Block forever
 	select {}
 }
+
+// firstVideoPayloadType finds the dynamic payload type of the first video
+// codec in the session description
+func firstVideoPayloadType(sd sdp.SessionDescription) (uint8, error) {
```

Comment:
I like this approach and I think this can stay here for now, but it might be good to move this to `sdp` eventually because it seems like it could be more generally useful.
The RFC says:
> If the <proto> sub-field is "RTP/AVP" or "RTP/SAVP" the <fmt>
      sub-fields contain RTP payload type numbers.  When a list of
      payload type numbers is given, this implies that all of these
      payload formats MAY be used in the session, but the first of these
      formats SHOULD be used as the default format for the session.

so we could code up this logic as `func (s *SessionDescription) GetDefaultPayloadType(mediaType string) (uint8, error) {` in `sdp/util.go` and use it for audio as well. Maybe create a mediaType enum while we're at it :wink:

Reply:
👍 I would prefer this as well. My goal  is be that users shouldn't import anything but `pion/webrtc` in the usual cases.

It sounds like there are even more issue we gotta deal with. Feel free to land anything as WIP. It can be frustrating to have long running PRs I don't want to burn you out.

Reply:
If you can get the build green, and then want to keep iterating on it.